src/parser/shared/modules/features/Checklist/ChecklistTypes.ts

```diff
@@ -0,0 +1,15 @@
+import Combatant from 'parser/core/Combatant';
+import CastEfficiency from 'parser/shared/modules/CastEfficiency';
+import { RequirementThresholds } from 'parser/shared/modules/features/Checklist/Requirement';
+
+export interface ChecklistProps {
+  combatant: Combatant,
+  castEfficiency: CastEfficiency,
+  thresholds: { [name: string]: RequirementThresholds }
+}
+
+export interface AbilityRequirementProps {
+  spell: number,
+  name?: string | JSX.Element,
```

Comment:
is there an example of where this is used? or can we just remove it?

Reply:
Once... 😓  in Brewmaster: 
``<AbilityRequirement spell={SPELLS.CELESTIAL_BREW.id} name={<><SpellLink id={SPELLS.CELESTIAL_BREW.id} /> cast efficiency</>} />
``